### PR TITLE
Extensions E2E tests

### PR DESCRIPTION
**Pull request by dechoD**
E2E tests for:
- Grid column
- Button in more actions
- Item title field
- Html field word count
- Html field Sitefinity videos

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -88,3 +88,6 @@ $RECYCLE.BIN/
 
 # Windows shortcuts
 *.lnk
+
+# Protractor test resuts
+e2e-tests.xml
```
Comment (jordanilchev) [inline .gitignore]: ↩️ ;)

File: tests/e2e/app-elements/item-details.component.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+require("jasmine-expect");
+import { ItemDetailsMap } from './item-details.map';
+import { BrowserWaitForElement, BrowserVerifyAlert } from "../helpers/browser-helpers";
+
+
```
Comment (jordanilchev) [inline tests/e2e/app-elements/item-details.component.ts]: duplicate empty lines

File: tests/e2e/app-elements/item-details.component.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+export class ItemDetails{
+
+    static async VerifyHtmlToolbarWordCount(): Promise<void> {
+        const wordCountButtonClass = "k-i-Words-count";
+        await BrowserWaitForElement(ItemDetailsMap.ToolbarButton(wordCountButtonClass));
+        var toolbarButton = await ItemDetailsMap.ToolbarButton(wordCountButtonClass);
```
Comment (jordanilchev) [inline tests/e2e/app-elements/item-details.component.ts]: do we need ```await``` here

File: tests/e2e/app-elements/item-list.component.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    private static async VerifyHeaders(headers: string[]) {
+        const headersNumber = headers.length;
+        try {
+            expect(await ItemListMap.TableHeaders.count()).toBe(headersNumber);
+            for (let i = 0; i < headersNumber; i++) {
+                const innerText = await ItemListMap.TableHeaders.get(i).getAttribute("innerText");
+                expect(innerText.toUpperCase().trim()).toBe(headers[i].toUpperCase());
+            }
+        } catch (e) {
+            throw new Error("Table headers are not correct.");
```
Comment (jordanilchev) [inline tests/e2e/app-elements/item-list.component.ts]: should we use throw here maybe failed expect will be more noticeable? @veliniv

File: tests/e2e/app-specs/verify-extensions.e2e-spec.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import { initAuth } from "../helpers/authentication-manager";
+import { ItemList } from '../app-elements/item-list.component';
```
Comment (jordanilchev) [inline tests/e2e/app-specs/verify-extensions.e2e-spec.ts]: We are supposed to use only double quotes per our guidelines.
@dechoD @veliniv maybe we are missing tslint

File: tests/e2e/headless-e2e.sh [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+createdUser=${BASH_REMATCH[1]}
+echo "UserName is: $createdUser"
+
+regex='password":"(.*)"'
+[[ $response =~ $regex ]]
+createdPass=${BASH_REMATCH[1]}
+echo "Password is: $createdPass"
+
+npm run e2e -- --params.sfUrl="$sandboxurl" --params.login.username="$createdUser" --params.login.password="$createdPass" --params.browser="chrome" --params.headless
+
```
Comment (jordanilchev) [inline tests/e2e/headless-e2e.sh]: unnecessary whitespaces

File: tests/e2e/helpers/authentication-manager.ts [hunk 7]
```diff
@@ -0,0 +1,171 @@
+/* tslint:disable:no-console */
+
```
Comment (jordanilchev) [inline tests/e2e/helpers/authentication-manager.ts]: unnecessary blank line

**Comment by mstratiev** [comment]
does this need "protractor" as a dependency?

**Pull request merged**